app/kuma-ui/pkg/server/server.go

```diff
@@ -10,20 +10,27 @@ import (
 	"github.com/Kong/kuma/pkg/core"
 	core_runtime "github.com/Kong/kuma/pkg/core/runtime"
 	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"strings"
 )
 
 var log = core.Log.WithName("gui-server")
 
 func SetupServer(rt core_runtime.Runtime) error {
-	srv := Server{rt.Config().GuiServer}
+	srv := Server{
+		Config:        rt.Config().GuiServer,
+		ApiServerPort: rt.Config().ApiServer.Port,
```

Comment:
Why `ApiServerPort` is not a part of `Config` ?

Reply:
Moved as a part of the configuration.